src/main/java/core/basesyntax/service/impl/FileServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataTransactionParser;
+import core.basesyntax.service.FileService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportGeneratorService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.impl.BalanceOperationImpl;
+import core.basesyntax.strategy.operation.impl.PurchaseOperationImpl;
+import core.basesyntax.strategy.operation.impl.ReturnOperationImpl;
+import core.basesyntax.strategy.operation.impl.SupplyOperationImpl;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FileServiceImpl implements FileService {
```

Comment:
```suggestion
public class FruitServiceImpl implements FruitService {
```
let's make this service to be responsible only for managing information about fruits, 
so we shouldn't have any reading or writing operations